common/djangoapps/util/enterprise_helpers.py

```diff
@@ -0,0 +1,22 @@
+"""
+Utility library for working with the edx-enterprise app
+"""
+
+from django.conf import settings
+
+
+def get_enterprise_branding_info(provider_id):
+    """
+    Client API operation adapter/wrapper
+    """
+    if not enterprise_enabled():
+        return None
+    from enterprise import api as enterprise_api
+    return enterprise_api.get_enterprise_branding_info(provider_id=provider_id)
+
+
+def enterprise_enabled():
+    """
+    Returns boolean indication if enterprise app is enabled on not.
+    """
+    return settings.FEATURES.get('ENTERPRISE_APP', True)
```

Comment:
@asadiqbal08  By the way, check out #14053  and specifically the new `utils.py` file, which adds an `enterprise_enabled` method. If that PR merges first, you'll want to update that method to use your new flag.